modules/models/src/main/java/org/eclipse/xpanse/modules/models/servicetemplate/ServiceFlavor.java

```diff
@@ -30,8 +31,8 @@ public class ServiceFlavor implements Serializable {
     private String name;
 
     @NotNull
-    @Schema(description = "The price of the flavor")
-    private Integer fixedPrice;
+    @Schema(description = "The pricing of the flavor")
+    private RatingMode pricing;
```

Comment:
I feel now that it is better we make a different type for ISV with the `pricing` and the existing ServiceFlavor without `pricing` for end users. What do you think about it?

Reply:
If there are some changes to `ServiceFlavor` in featrue, it may not be good for us to maintain two models. If you don't want to return null, we can set the objectMapper configuration.

Reply:
@baixinsui My only concern is, if the end user sees the data model, he sees that there is a "price" field there. so returning null or removing the field completely does not matter.